reana-ui/src/pages/profile/components/Quota.js

```diff
@@ -0,0 +1,68 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React from "react";
+import { PieChart } from "react-minimal-pie-chart";
+import { Grid, Label } from "semantic-ui-react";
+
+import styles from "./Quota.module.scss";
+
+const SEPIA_COLOR = "#f5ecec";
+const DARK_SEPIA_COLOR = "#b68181";
+
+export default function Quota() {
+  const dummy = [
+    { title: "CPU", value: 71, limit: 120, unit: "min" },
+    { title: "HDD", value: 83232, limit: 123456, unit: "KiB" },
+  ];
+  return (
+    <Grid columns={2}>
+      {dummy.map(({ title, value, limit, unit }) => {
+        const percentage = ((value / limit) * 100).toFixed(2);
+        let percentageColor = "green";
+        // TODO: To be consistent with `reana-client quota-show`
+        if (percentage >= 60) {
+          if (percentage >= 85) {
+            percentageColor = "red";
+          } else {
+            percentageColor = "orange";
+          }
+        }
+        return (
+          <Grid.Column className={styles.column}>
+            <PieChart
+              data={[{ title: title, value: value, color: DARK_SEPIA_COLOR }]}
+              background={SEPIA_COLOR}
+              totalValue={limit}
```

Comment:
Now that we have implemented that `quota_limit = 0` means no limit. How can we represent this with pie charts? I'm afraid they don't make sense in the case.